### PR TITLE
[JENKINS-70169] Add missing breadcrumb items in various `resources/hudson` locations

### PR DESCRIPTION
**Pull request by lemeurherve**
Extract from #7457 (remaining from the other PRs, too few on each folder to merit its own PR)

Follow-up of https://github.com/jenkinsci/jenkins/pull/6912

Initially fixing [JENKINS-70169](https://issues.jenkins.io/browse/JENKINS-70169), I've included every missing breadcrumb items I've found by searching for "sidepanel.jelly" in jelly files.

Notes: 
- I've added `<!-- no need for breadcrumb here -->` as comment in views where adding a breadcrumb item wasn't necessary, let me know if it's preferable to omit them.

- This doesn't fix missing breadcrumb depending on plugin (ex: https://github.com/jenkinsci/cloudbees-folder-plugin/pull/278)

### Testing done

Built and ran Jenkins locally, going on modified pages to check the change.

There is no additional automatic test added for these breadcrumb items.

With a job "test",

`http://localhost:8080/jenkins/job/test/ws/`:
image

`http://localhost:8080/jenkins/job/test/buildTimeTrend`:
image

`http://localhost:8080/jenkins/user/test/my-views/newView`:
image

`http://localhost:8080/jenkins/signup/`,
Before, no link on "Dashboard", and no side panel:
image

After:
image

I'm not sure where to find the following corresponding changes (thus untested and potentially not wanted):
- core/src/main/resources/hudson/cli/CLIAction/command.jelly
- core/src/main/resources/hudson/model/Label/load-statistics.jelly
- core/src/main/resources/hudson/model/labels/LabelAtom/configure.jelly

### Proposed changelog entries

- Add missing breadcrumb items in various locations.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] The Jira issue, if it exists, is well-described.
- [X] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [X] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7496"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/hudson/model/labels/LabelAtom/configure.jelly
Comment (NotMyFault) [reply core/src/main/resources/hudson/model/labels/LabelAtom/configure.jelly:33]: That looks better.

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
> `http://localhost:8080/jenkins/signup/`, Before, no link on "Dashboard", and no side panel: image
> 
> After: image

Caused [SECURITY-3594](https://www.jenkins.io/security/advisory/2025-09-17/#SECURITY-3594) (with assist from me in https://github.com/jenkinsci/jenkins/pull/1388)

**Comment by lemeurherve** [comment]
Oh... 😓 🤦

**Comment by daniel-beck** [comment]
This shouldn't even have been possible though. Fixed https://github.com/jenkinsci/jenkins/commit/e5fb48fd045b0a94c086707e1fe698c143a681ef?w=1#diff-d228ecbbc67a084eff38f5d8af678f7ee11df824773e7962baff47aa46265309 now.

**Pull request merged**
